spec/helpers.rb

```diff
@@ -2,6 +2,15 @@ def auth_user_headers
   user.create_new_auth_token
 end
 
+def another_auth_user_headers
+  another_user.create_new_auth_token
+end
+
 def parsed_data
   JSON.parse(response.body)
 end
+
+def conversation_user
```

Comment:
Maybe you can create a scope in the model, just a suggestion to avoid this. Also it could be helpful in a future.